modifiers.py

```diff
@@ -24,4 +23,14 @@ def stun(self):
 
 
 class Poison(Modifier):
-    pass
+    def act(self):
+        self.poison()
+
+    def poison(self):
+        if self.duration > 0:
+            self.target.current_hp -= self.damage
+            self.duration -= 1
+            print(f"poisoned, damage {self.damage}, duration: {self.duration}")
+        else:
+            self.target.next_move = self.target.act
```

Comment:
Nie nie nie!
To był zabieg, który pozwalał na zakończenia stunu. Tu coś takiego nie powinno się zdarzyć. W tym wydaniu koniec bycia otrutym zdejmuje efekt zestunowania z bohatera..

Reply:
ok done